scripts/packaging/createupmpackages.ps1

```diff
@@ -159,11 +159,29 @@ foreach ($entry in $packages.GetEnumerator()) {
         $registryName = "the $OutputTarget registry"
     }
 
-    # The examples folder needs a custom setup step
-    if ($entry.Name -eq "examples") {
-        Write-Output "Preparing the examples folder"
+    $samplesFolder = "$packagePath\Samples~"
+     
+    if ($packageName -eq "examples") {
+        # The examples folder needs a custom setup step
         Start-Process -FilePath "$PSHOME\powershell.exe" -ArgumentList "$scriptPath\examplesfolderpreupm.ps1" -NoNewWindow -Wait
     }
+    else {
+        # Some other folders have localized examples that need to be prepared. Intentionally skip the foundation as those samples
+        # are pacakged in the examples package.
+        $exampleFolder = "$packagePath\Examples"
+        if (($PackageName -ne "foundation") -and (Test-Path -Path $exampleFolder)) {
+            if (Test-Path -Path $exampleFolder) {

```

Comment:
I think this clause is already in the if statement above (so this line can be deleted)

Reply:
will double check

Reply:
sharp eyes! thanks!

Reply:
fixed